JiraReporter/AuthorsProcessing.cs

```diff
@@ -0,0 +1,383 @@
+﻿using JiraReporter.Model;
+using System;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Text;
+using System.Text.RegularExpressions;
+using RestSharp;
+
+namespace JiraReporter
+{
+    class AuthorsProcessing
```

Comment:
Fisierul asta noi l-am modificat, nu?

Cred ca trebuie sa faci merge din master in branch.